server/athenian/api/models/web/calculated_pull_request_histogram.py

```diff
@@ -55,15 +55,15 @@ def __init__(
         self._interquartile = interquartile
 
     @property
-    def for_(self) -> ForSet:
+    def for_(self) -> ForSetPullRequests:
```

Comment:
I considered this right and typing in `__init__` wrong according to discussion here
https://github.com/athenianco/athenian-api/blob/master/server/athenian/api/controllers/metrics_controller.py#L87

Reply:
`ForSetPullRequests` is a type variable so it's not valid for mypy (like `ForSetCodeChecks`) but this is not obvious to fix